File: tessoku/C++/chap01/A03.cpp
#include <iostream>
using namespace std;

int N, K, P[109], Q[109];
bool ans = false;

int main() {
    cin >> N >> K;
    for (int i = 1; i <= N; i++) cin >> P[i];
    for (int i = 1; i <= N; i++) cin >> Q[i];

    for (int x = 1; x <= N; x++) {
        for (int y = 1; y <= N; y++) {
            if (P[x] + Q[y] == K) ans = true;
        }
    }

    if (ans == true) cout << "Yes" << endl;
    else cout << "No" << endl;

    return 0;
}

File: tessoku/Python/chap01/A03.py
N, K = map(int, input().split())
P = list(map(int, input().split()))
Q = list(map(int, input().split()))

ans = False
for p in P:
    for q in Q:
        if (p + q) == K:
            ans = True
            break 

if ans == True:
    print('Yes')
else:
    print('No')